crates/gitbutler-project/src/storage.rs

```diff
@@ -28,6 +28,7 @@ pub struct UpdateRequest {
     pub use_diff_context: Option<bool>,
     pub snapshot_lines_threshold: Option<usize>,
     pub succeeding_rebases: Option<bool>,
+    pub git_host: Option<GitHostSettings>,
```

Comment:
This struct should ideally also have the camelCase transformation, but outside the scope of this pr.